sources/platform/actors/limits.md

```diff
@@ -1,50 +1,134 @@
-| Build timeout                                                    | 1800 secs<!-- ACTOR_LIMITS.BUILD_TIMEOUT_SECS -->                                     |
-| Build/run disk size                                              | 2x job memory limit<!-- ACTOR_LIMITS.RUN_DISK_TO_MEMORY_SIZE_COEFF -->                |
-| Memory per CPU core                                              | 4,096 MB<!-- ACTOR_LIMITS.RUN_MEMORY_MBYTES_PER_CPU_CORE -->                          |
-| Build/run maximum log size                                       | 10,485,760 characters<!-- ACTOR_LIMITS.LOG_MAX_CHARS -->                              |
-| Maximum number of dataset columns for xlsx format                | 2,000 columns                                                                         |
-| Maximum size of input schema for a task/actor                    | 100 kB<!-- ACTOR_LIMITS.INPUT_SCHEMA_MAX_BYTES -->                                    |
-
-## Platform limits {#platform-limits}
-
-| Description                                                        | Value                                                                               |
-|--------------------------------------------------------------------|-------------------------------------------------------------------------------------|
-| Maximum number of actors per user                                  | 100<!-- DEFAULT_PLATFORM_LIMITS.MAX_ACTORS_PER_USER -->                             |
-| Maximum number of tasks per user                                   | 1000<!-- DEFAULT_PLATFORM_LIMITS.MAX_TASKS_PER_USER -->                             |
-| Maximum number of schedules per user                               | 100<!-- DEFAULT_PLATFORM_LIMITS.MAX_SCHEDULES_PER_USER -->                          |
-| Maximum number of webhooks per user                                | 100<!-- DEFAULT_PLATFORM_LIMITS.MAX_TASKS_PER_USER -->                              |
-| Maximum number of concurrent actor runs per user for free accounts | 25<!-- DEFAULT_PLATFORM_LIMITS.FREE_ACCOUNT_MAX_CONCURRENT_ACTOR_RUNS_PER_USER -->  |
-| Maximum number of concurrent actor runs per user for paid accounts | 250<!-- DEFAULT_PLATFORM_LIMITS.PAID_ACCOUNT_MAX_CONCURRENT_ACTOR_RUNS_PER_USER --> |
-| Maximum number of actors per schedule                              | 10<!-- DEFAULT_PLATFORM_LIMITS.MAX_ACTORS_PER_SCHEDULER -->                         |
-| Maximum number of tasks per schedule                               | 10<!-- DEFAULT_PLATFORM_LIMITS.MAX_TASKS_PER_SCHEDULER -->                          |
+The tables below demonstrate the Apify Platform's default resource limits. For API limits such as rate limits and max payload size, see the [API documentation](https://docs.apify.com/api/v2#/introduction/rate-limiting).
+
+> If needed, the limits shown below can be increased on paid accounts. For details, contact us on **hello@apify.com** or using the chat icon in the bottom-right corner.
```

Comment:
I don't think there's a chat icon in the bottom right corner anymore. Or does it automatically appear on the docs when you're a paying user?